packages/block-library/src/comments-pagination/editor.scss

```diff
@@ -5,7 +5,7 @@ $pagination-margin: 0.5em;
 	justify-content: center;
 }
 
-.editor-styles-wrapper {
+:where(.editor-styles-wrapper) {
```

Comment:
I decided to lower the specificity here, though I can't see that these styles are doing anything. 🤔 

Same applies for query pagination.

Reply:
> I don't know if that was the intention

Sounds like it was unintentional. The original PR for the block doesn't give any clues either.